--- prisma/scheme.prisma	2022-10-02 21:46:25.330475821 +0300
+++ prisma/schema.prisma	2022-10-02 21:45:22.822031996 +0300
@@ -3,21 +3,21 @@
 }
 
 datasource db {
-  provider = "mysql"
+  provider = "mongodb"
   url      = env("DATABASE_URL")
 }
 
 model User {
-  id           String    @id @default(uuid())
-  username     String    @unique @db.VarChar(64)
-  email        String    @db.VarChar(320)
-  password     String?   @db.VarChar(60)
-  secret2fa    String?   @db.VarChar(20)
-  avatar       String?   @db.Text
-  banner       String?   @db.Text
+  id           String    @id @default(auto()) @map("_id") @db.ObjectId
+  username     String    @unique 
+  email        String    
+  password     String?   
+  secret2fa    String?   
+  avatar       String?
+  banner       String?
   birthday     DateTime? @db.Date
   gender       Gender    @default(UNSPECIFIED)
-  customGender String?   @db.VarChar(64)
+  customGender String?   
   createdAt    DateTime  @default(now())
   deleted      Boolean   @default(false)
 
@@ -25,9 +25,9 @@
 }
 
 model SiteAuthSession {
-  id     String  @id @default(uuid())
-  agent  String  @db.Text
-  ip     String  @db.VarChar(64)
+  id     String  @id @default(auto()) @map("_id") @db.ObjectId
+  agent  String
+  ip     String 
   userId String
   user   User    @relation(fields: [userId], references: [id])
   active Boolean @default(true)
